### PR TITLE
Fix Docs Browser&Menu&Page Title Differences

### PR DESCRIPTION
**Pull request by salihozkara**
Resolves https://github.com/volosoft/vs-internal/issues/4854

### CODE REVIEW

**Review by ebicoglu — APPROVED** [review]

**Review by oykuermann — APPROVED** [review]

File: docs/en/docs-nav.json [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-          "text": "Microservice solution",
+          "text": "Microservice Solution",
           "path": "get-started/microservice.md"
         },
         {
           "text": "Others",
           "items": [
             {
-              "text": "Empty ASP.NET Core application",
+              "text": "Empty ASP.NET Core MVC / Razor Pages Application",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:26]: Revert that please.

File: docs/en/docs-nav.json [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
               "path": "get-started/empty-aspnet-core-application.md"
             },
             {
-              "text": "MAUI application",
+              "text": "MAUI Application Startup Template",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:30]: MAUI Application

File: docs/en/docs-nav.json [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
               "path": "get-started/maui.md"
             },
             {
-              "text": "WPF application",
+              "text": "WPF Application Startup Template",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:34]: WPF Application

File: docs/en/docs-nav.json [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
               "path": "get-started/wpf.md"
             },
             {
-              "text": "Console application",
+              "text": "Console Application Startup Template",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:38]: Console Application

File: docs/en/docs-nav.json [hunk 5]
```diff
@@ -57,60 +57,60 @@
...
             {
-              "text": "Layered solution",
+              "text": "Layered Solution",
               "path": "tutorials/todo/layered"
             }
           ]
         },
         {
-          "text": "Book Store Application",
+          "text": "Web Application Development",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:70]: Revert this please.

File: docs/en/docs-nav.json [hunk 6]
```diff
@@ -653,7 +653,7 @@
               "path": "framework/architecture"
             },
             {
-              "text": "Best Practices",
+              "text": "Module Development Best Practices & Conventions",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:656]: Module Development Best Practices

File: docs/en/docs-nav.json [hunk 7]
```diff
@@ -1631,7 +1631,7 @@
           "path": "solution-templates/microservice"
         },
         {
-          "text": "Application Module",
+          "text": "Module Solution",
```
Comment (hikalkan) [inline docs/en/docs-nav.json:1634]: Revert please.

**Pull request merged**